src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/PlayersController.cs

```diff
@@ -333,5 +333,19 @@ private static List<int> ParseIntList(string source)
 
             return result;
         }
+
+        private bool IsFreePlayer(Player player, int? includeTeam)
+        {
+            var teamId = _playerService.GetPlayerTeam(player).Id;
+
+            if (includeTeam.HasValue)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the '?:' operator here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3240)